test/migrations/namespace_get_function.hrx

```diff
@@ -0,0 +1,37 @@
+<==> input/entrypoint.scss
+@import "library";
+
+$fn1: get-function(increment);
+$fn2: get-function($name: "increment");
+$fn3: get-function("incre" + "ment");
+
+$x: "increment";
+$fn4: get-function($x);
+
+<==> input/_library.scss
+@function increment($x) {
+  @return $x + 1;
+}
+
+<==> output/entrypoint.scss
+@use "sass:meta";
+@use "library";
+
+$fn1: meta.get-function(increment, $module: library);
+$fn2: meta.get-function($name: "increment", $module: library);
+$fn3: meta.get-function("incre" + "ment");
+
+$x: "increment";
+$fn4: meta.get-function($x);
+
+<==> log.txt
+line 5, column 20 of $TEST_DIR/entrypoint.scss: WARNING - get-function call may require $module parameter
+  ,
+5 | $fn3: get-function("incre" + "ment");
+  |                    ^^^^^^^^^^^^^^^^
+  '
+line 8, column 20 of $TEST_DIR/entrypoint.scss: WARNING - get-function call may require $module parameter
+  ,
+8 | $fn4: get-function($x);
+  |                    ^^
+  '
```

Comment:
Also test a getting a function from a core library when the function's namespaced name doesn't match its original name, and getting a function from a library named `true` or whatever.